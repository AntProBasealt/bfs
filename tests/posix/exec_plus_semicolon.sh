# POSIX says:
#     Only a <plus-sign> that immediately follows an argument containing only the two characters "{}"
#     shall punctuate the end of the primary expression. Other uses of the <plus-sign> shall not be
#     treated as special.
bfs_diff basic -exec echo foo {} bar + baz \;
