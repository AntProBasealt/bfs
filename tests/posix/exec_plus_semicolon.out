foo basic bar + baz
foo basic/a bar + baz
foo basic/b bar + baz
foo basic/c bar + baz
foo basic/c/d bar + baz
foo basic/e bar + baz
foo basic/e/f bar + baz
foo basic/g bar + baz
foo basic/g/h bar + baz
foo basic/i bar + baz
foo basic/j bar + baz
foo basic/j/foo bar + baz
foo basic/k bar + baz
foo basic/k/foo bar + baz
foo basic/k/foo/bar bar + baz
foo basic/l bar + baz
foo basic/l/foo bar + baz
foo basic/l/foo/bar bar + baz
foo basic/l/foo/bar/baz bar + baz
